cypress/e2e/environment.cy.js

```diff
@@ -0,0 +1,389 @@
+          break;
+      }
+      if (parameter.name !== "Repository" && parameter.name !== "Branch") {
+        cy.get('[aria-label="' + parameter.name + '-submit-edit"]').click();
+        cy.get('[aria-label="' + parameter.name + '-value"]').should(
+          "contain",
+          "New Value " + parameter.name
+        );
+      } else {
+        cy.get('[aria-label="Repository Settings-submit-edit"]').click();
+        if (parameter.name === "Repository") {
+          cy.get('[aria-label="repo_url-value"]').should(
+            "contain",
+            "New Value " + parameter.name
+          );
+        } else {
+          cy.get('[aria-label="repo_branch-value"]').should(
+            "contain",
+            "New Value " + parameter.name
+          );
+        }
+      }
+    });
+    deleteEnv(4);
+  });
+
+  it("1.5 Clear environment", function () {
+    //Fill The form and submit
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("test").click();
+    cy.get('[aria-label="ServiceCatalog-Success"]').should("to.be.visible");
+
+    //Go to settings
+    cy.get('[aria-label="Settings actions"]').click();
+    cy.url().should("contain", "/console/settings?env=");
+    cy.get('[aria-label="Name-value"]').should("contain", "test");
+
+    //Cancel Clear Env and expect nothing to change
+    cy.get("button").contains("Clear environment").click();
+    cy.get("button").contains("Cancel").click();
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("test").click();
+    cy.get('[aria-label="ServiceCatalog-Success"]').should("to.be.visible");
+
+    //Go to settings
+    cy.get('[aria-label="Settings actions"]').click();
+    cy.url().should("contain", "/console/settings?env=");
+    cy.get('[aria-label="Name-value"]').should("contain", "test");
+
+    //Clear Env
+    cy.get("button").contains("Clear environment").click();
+    cy.get('[aria-label="clear environment check"]').type("test");
+    cy.get("button")
+      .contains("I understand the consequences, clear this environment")
+      .click();
+    cy.visit("/console/");
+    cy.get('[aria-label="Environment card"]').contains("test").click();
+    cy.get('[aria-label="ServiceCatalog-Empty"]').should("to.be.visible");
+    //Update service catalog to restore instances
+    cy.get("button").contains("Update Service Catalog").click();
+    cy.get("button").contains("Yes").click();
+    cy.intercept(
+      "/api/v2/environment/a5f9f7f2-5f43-4fad-a16d-f71f127f0e91?details=fals"
+    ).as("catalogUpdate");
+    cy.wait("@catalogUpdate")
+      .its("response.body")
+      .then(() => {
+        cy.get('[aria-label="ServiceCatalog-Success"]').should("to.be.visible");
+      });
+  });
+
+  it("1.6 Edit environment configuration", function () {
+    cy.visit("/console/environment/create");
+    createEnv(6, true, true);
+    cy.get("button").contains("Submit").click();
+    cy.url().should("contain", "/console/lsm/catalog?env=");
+
```

Comment:
you added a comment in previous steps ;) would be nice to keep this :)

Reply:
That's the plan :D I left it when all tests will be properly set up